pycolmap/pycolmap/log_exceptions.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
```

Comment:
As a follow-up, we should probably move this file to colmap and replace any glog CHECK* macros with THROW_CHECK*? I can try to look into this as a follow-up.

Reply:
Yes definitely, will address this in a later PR.